x-pack/plugins/enterprise_search/public/applications/app_search/components/credentials/credentials.tsx

```diff
@@ -68,35 +64,44 @@ export const Credentials: React.FC = () => {
       </EuiPageHeader>
       <EuiPageContentBody>
         <EuiPanel className="eui-textCenter">
-          <EuiTitle size="s">
-            <h2>
-              {i18n.translate('xpack.enterpriseSearch.appSearch.credentials.apiEndpoint', {
-                defaultMessage: 'Endpoint',
-              })}
-            </h2>
-          </EuiTitle>
-          <EuiCopy
-            textToCopy={externalUrl.enterpriseSearchUrl}
-            afterMessage={i18n.translate('xpack.enterpriseSearch.appSearch.credentials.copied', {
-              defaultMessage: 'Copied',
-            })}
-          >
-            {(copy) => (
-              <>
-                <EuiButtonIcon
-                  onClick={copy}
-                  iconType="copyClipboard"
-                  aria-label={i18n.translate(
-                    'xpack.enterpriseSearch.appSearch.credentials.copyApiEndpoint',
-                    {
-                      defaultMessage: 'Copy API Endpoint to clipboard.',
-                    }
-                  )}
-                />
-                {externalUrl.enterpriseSearchUrl}
-              </>
-            )}
-          </EuiCopy>
+          {!!dataLoading ? (
+            <EuiLoadingContent lines={3} />
+          ) : (
```

Comment:
Question on this - do we really want to show a loading skeleton for the Endpoint panel since it's not really showing any dynamic info? The endpoint is available immediately on page load, maybe we should just opt to show it immediately?

Reply:
Ah, I just assumed we had to wait for it to be available. But you're right, the plugin won't load until that data is fetched, so we can lose this.

Reply:
https://github.com/elastic/kibana/pull/80035/commits/9376d54d40f2caebc7050fa1d48ba2bb92c45d27?w=1